bridges/evm/contracts/Bridge.sol

```diff
@@ -135,66 +136,16 @@ contract Bridge is IBridge, Ownable {
         bytes32 dataHash;
     }
 
-    // /// Decodes the encoded result and returns back the decoded data which is the data and its context.
-    // /// @param _encodedData The encoded of result and its context.
-    // function decodeResult(bytes memory _encodedData)
-    //     public
-    //     pure
-    //     returns (VerifyOracleDataResult memory)
-    // {
-    //     require(_encodedData.length > 40, "INPUT_MUST_BE_LONGER_THAN_40_BYTES");
-
-    //     VerifyOracleDataResult memory result;
-    //     assembly {
-    //         mstore(
-    //             add(result, 0x20),
-    //             and(mload(add(_encodedData, 0x08)), 0xffffffffffffffff)
-    //         )
-    //         mstore(
-    //             add(result, 0x40),
-    //             and(mload(add(_encodedData, 0x10)), 0xffffffffffffffff)
-    //         )
-    //         mstore(
-    //             add(result, 0x60),
-    //             and(mload(add(_encodedData, 0x18)), 0xffffffffffffffff)
-    //         )
-    //         mstore(
-    //             add(result, 0x80),
-    //             and(mload(add(_encodedData, 0x20)), 0xffffffffffffffff)
-    //         )
-    //         mstore(
-    //             add(result, 0xa0),
-    //             and(mload(add(_encodedData, 0x28)), 0xffffffffffffffff)
-    //         )
-    //     }
-
-    //     bytes memory data = new bytes(_encodedData.length - 40);
-    //     uint256 dataLengthInWords = ((data.length - 1) / 32) + 1;
-    //     for (uint256 i = 0; i < dataLengthInWords; i++) {
-    //         assembly {
-    //             mstore(
-    //                 add(data, add(0x20, mul(i, 0x20))),
-    //                 mload(add(_encodedData, add(0x48, mul(i, 0x20))))
-    //             )
-    //         }
-    //     }
-    //     result.data = data;
-
-    //     return result;
-    // }
-
     /// Verifies that the given data is a valid data on BandChain as of the given block height.
     /// @param _blockHeight The block height. Someone must already relay this block.
-    /// @param _data The data to verify, with the format similar to what on the blockchain store.
-    /// @param _requestId The ID of request for this data piece.
+    /// @param _requestPacket The request packet contain detail of request.
+    /// @param _responsePacket The response packet of this request contain metadata of result.
```

Comment:
Let's remove "contain...." part. Just say The request packet is this request, The response packet of this request.